### ISSUE TITLE
Test AMI with METplus experts

### ISSUE THREAD

**Issue by ian-noaa** [NOAA-GSL/VxMETplusAMI#25]
Get together with Tina Kalb to make sure METplus in the AMI performs as expected and the data is where we need it.

**Comment by ian-noaa** [comment]
The initial run-through of Tina's S2S Weather Regime analysis went well. We successfully ran the `UserScript_fcstGFS_obsERA_OMI.conf` and `UserScript_fcstGFS_obsERA_WeatherRegime.conf` use cases in `METplus/parm/use_cases/...`. 

We're now doing a long-running test - instructions for that can be found here: https://docs.google.com/document/d/1A_SZHPB-M1QV1bFzufbGoYVDvYKTfpLPYr6iOERDEdY/edit

We did discover we needed to add METcalcpy, METdatadb, and METplotpy to the `PYTHONPATH` variable. So this will need to be added to the AMI.
